Chummer/frmNotes.Designer.cs

```diff
@@ -28,31 +28,125 @@ protected override void Dispose(bool disposing)
+            this.txtNotes.Location = new System.Drawing.Point(3, 3);
             this.txtNotes.MaxLength = 2147483647;
             this.txtNotes.Multiline = true;
             this.txtNotes.Name = "txtNotes";
             this.txtNotes.ScrollBars = System.Windows.Forms.ScrollBars.Vertical;
-            this.txtNotes.Size = new System.Drawing.Size(624, 321);
-            this.txtNotes.TabIndex = 0;
-            this.txtNotes.KeyDown += new System.Windows.Forms.KeyEventHandler(this.txtNotes_KeyDown);
+            this.txtNotes.Size = new System.Drawing.Size(618, 281);
+            this.txtNotes.TabIndex = 1;
+            this.txtNotes.TextChanged += new System.EventHandler(this.txtNotes_TextChanged);
+            // 
+            // btnColorSelect
+            // 
+            this.btnColorSelect.Anchor = ((System.Windows.Forms.AnchorStyles)((System.Windows.Forms.AnchorStyles.Bottom | System.Windows.Forms.AnchorStyles.Left)));
+            this.btnColorSelect.AutoSize = true;
+            this.btnColorSelect.AutoSizeMode = System.Windows.Forms.AutoSizeMode.GrowAndShrink;
+            this.btnColorSelect.Image = global::Chummer.Properties.Resources.color_wheel;
+            this.btnColorSelect.ImageAlign = System.Drawing.ContentAlignment.TopLeft;
+            this.btnColorSelect.Location = new System.Drawing.Point(3, 295);
+            this.btnColorSelect.Name = "btnColorSelect";
+            this.btnColorSelect.Size = new System.Drawing.Size(190, 23);
+            this.btnColorSelect.TabIndex = 2;
+            this.btnColorSelect.Tag = "Button_ChooseCustomColor";
+            this.btnColorSelect.Text = "Choose custom color for this note";
+            this.btnColorSelect.TextAlign = System.Drawing.ContentAlignment.MiddleLeft;
+            this.btnColorSelect.TextImageRelation = System.Windows.Forms.TextImageRelation.ImageBeforeText;
+            this.btnColorSelect.UseVisualStyleBackColor = true;
+            this.btnColorSelect.Click += new System.EventHandler(this.btnColorSelect_Click);
+            // 
+            // bufferedTableLayoutPanel2
+            // 
+            this.bufferedTableLayoutPanel2.ColumnCount = 2;
+            this.bufferedTableLayoutPanel2.ColumnStyles.Add(new System.Windows.Forms.ColumnStyle());
+            this.bufferedTableLayoutPanel2.ColumnStyles.Add(new System.Windows.Forms.ColumnStyle());
+            this.bufferedTableLayoutPanel2.Controls.Add(this.btnOK, 1, 0);
+            this.bufferedTableLayoutPanel2.Controls.Add(this.btnCancel, 0, 0);
+            this.bufferedTableLayoutPanel2.Location = new System.Drawing.Point(477, 290);
+            this.bufferedTableLayoutPanel2.Name = "bufferedTableLayoutPanel2";
+            this.bufferedTableLayoutPanel2.RowCount = 1;
+            this.bufferedTableLayoutPanel2.RowStyles.Add(new System.Windows.Forms.RowStyle());
+            this.bufferedTableLayoutPanel2.RowStyles.Add(new System.Windows.Forms.RowStyle(System.Windows.Forms.SizeType.Absolute, 29F));
```

Comment:
You probably don't need two rows for this panel.

Reply:
Indeed. The second row was not even visible in the graphical designed, seems like it's only the style. Will be removed in next revision.